ament_clang_format/package.xml

```diff
@@ -9,6 +9,11 @@
   <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
   <license>Apache License 2.0</license>
 
+  <build_depend>python3-setuptools</build_depend>
+
+  <buildtool_depend>python3</buildtool_depend>
```

Comment:
Do we really need to tell a Python package that it needs `python3` to invoke the `setup.py`? It looks superfluous to me (I made the same comments on similar PRs).